doc/api/zlib.markdown

```diff
@@ -366,11 +367,79 @@ Decompress a Buffer or string with InflateRaw.
 
 Decompress a Buffer or string with Unzip.
 
+## Class GzipHeader
+
+<!--type=misc-->
+
+The zlib bindings support reading and writing fields of the Gzip file format
+header. A Gzip header has the following properties:
+
+* `text` {Boolean}
+* `time` {Integer|Date}
+* `os` {Integer}
+* `hcrc` {Boolean}
+* `extra` {Array|Buffer|Null}
+* `comment` {String|Buffer|Null}
+* `name` {String|Buffer|Null}
+
+`extra` can be a {Buffer} or a list of sub-fields, each an object with a `id`
+and a `content` property as per [RFC1952][]. More information on the meaning
+of these fields is available at <http://zlib.net/manual.html#Advanced>.
```

Comment:
Perhaps a better (or at least an additional) link here would be to the [gzip rfc](http://www.zlib.org/rfc-gzip.html)?